plugins/mumble_plugin_win32.h

```diff
@@ -105,16 +154,25 @@ static bool inline initialize(const std::multimap<std::wstring, unsigned long lo
 	if (!dwPid)
 		return false;
 
+#ifndef USE_PLUGIN_LEGACY_PTR
+	static VOID *pModule;
+#endif
 	pModule=getModuleAddr(modname ? modname : procname);
 	if (!pModule) {
 		dwPid = 0;
 		return false;
 	}
+	pModule32 = reinterpret_cast<uintptr32_t>(pModule);
```

Comment:
Actually, it *does* make very much sense...

Consider a game that ships:

    x86/game.exe
    x64/game.exe

Both games are looked up with "game.exe"... A game like that would need a unified plugin, and determine which exe to use by checking which of pModule32 or pModule64 is non-NULL.